app/services/rapid_ready_for_decision/fast_track_pdf_generator.rb

```diff
@@ -24,13 +25,27 @@ def self.extract_patient_name(patient_info)
       [full_name, patient_info[:suffix]].reject(&:blank?).join ', '
     end
 
+    # TODO: move this into the data class
+    def self.flag_with_keyword(keywords, medication_struct)
+      keywords.any? { |keyword| medication_struct.to_s.downcase.include?(keyword.downcase) }
```

Comment:
Instead of `keyword.downcase`, I was thinking you can downcase the `keywords` in the `Constants` class.